src/Func.h

```diff
@@ -100,6 +177,7 @@ class Stage {
         return reorder(collected_args);
     }
 
+    EXPORT Stage &purify(VarOrRVar old_name, VarOrRVar new_name);
```

Comment:
Probably shouldn't expose this to the front-end, because it's possible to inject race conditions using it.